.github/workflows/ci-windows.yml

```diff
@@ -66,6 +66,9 @@ jobs:
       if: steps.cache-ext.outputs.cache-hit != 'true'
       working-directory: ./ext
       run: ./libyuv.cmd
+      env:
+        CC: clang-cl
+        CXX: clang-cl
```

Comment:
I believe `.github/workflows/ci-windows.yml` was made to test MSVC compilation success rather than test performance. I recommend reverting this file.

This is a good idea for AppVeyor though, because its purpose is binary distribution for users.

Reply:
Done.

Reply:
I just saw this discussion. I think this change is fine because libyuv is less valuable without its assembly code. I wanted to make this kind of change but I didn't have time to look into it.

Yuan: Because of the use of the cache files for the `ext/` directory, you will need to make a dummy change to one of the `ext/*cmd` files in this pull request, or temporarily comment out line 66 `if: steps.cache-ext.outputs.cache-hit != 'true'`.

Reply:
Just to clarify -- it is okay to deter this change to a separate pull request.

Reply:
> Yuan: Because of the use of the cache files for the `ext/` directory, you will need to make a dummy change to one of the `ext/*cmd` files in this pull request, or temporarily comment out line 66 `if: steps.cache-ext.outputs.cache-hit != 'true'`.

I wonder if an alternative in changes like this would be to add a version (v1,v2,etc) to the `key: ${{ runner.os }}-${{ hashFiles('ext/*.cmd') }}` and increase it to force a rerun.

Reply:
Yuan: I just noticed that this pull request contains a change to `ext/libyuv.cmd`, so no need for a dummy change to one of the `ext/*cmd` files. Sorry about my confusion.

James: I am considering adding a dummy file such as `ext/touch.cmd` or `ext/update-cache.cmd`